### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by mattheu** [humanmade/hm-pattern-library#91]
Some accessibility concerns have been raised, I'm creating a big issue for all of them here. Feel free to edit this and add more tasks. 
- [ ] The secondary blue buttons with white text do not meet contrast accessibility even for large text. I'm adding some screenshots of how it would look if we tried to meet at least WACG AA for large text below. Personally, I would like to avoid changing the shade of blue, as this is a brand color, and we're pretty committed on that side, but also I'm not a fan of the dark text on the buttons, but maybe I'm just to close to the original design to be objective. Also, it would be a shame to loose the blue across the site as I think it really breaks up the red which is a very dominant colour when used on its own.
- [ ] The HM red brand colour is used for link text color, but at 18px it doesn't meet WACG AA guidelines for contrast. Its ok for larger text ([18pt+ according to this tool](https://snook.ca/technical/colour_contrast/colour.html#fg=CA4A37,bg=FFFFFF) but what does that mean in px sizes? @rianrietveld said 19px) 
- [ ] Jenny raised some concerns about all caps for headings/menus/buttons over here: https://github.com/humanmade/hm-pattern-library/issues/75

**Other design issues**
- [ ] Links presentation : not by colour alone
- [ ] Better .focus style for buttons
- [ ] Colour contrast placeholder needs improvement
See remarks below

Obviously, we'd like to be as committed to accessibility as we can be, so I'd like to try and find a solution to this. One thing to note, there will be some big changes coming to the pattern library when we update it to fit in with the new HMN website redesign. Probably our best hope of fixing this is to feed back to barbara so she can work on that with this in mind.

![image](https://cloud.githubusercontent.com/assets/494927/19801650/03171ec2-9cf8-11e6-83da-381c9ebbfceb.png)

![image](https://cloud.githubusercontent.com/assets/494927/19801681/266a202c-9cf8-11e6-9b9b-360274070255.png)

**Comment by mattheu** [comment]
CC @grarighe

**Comment by rianrietveld** [comment]
I've done an accessibility review on the patten library for level WCAG 2 AA 
I found the following issues next to the ones already mentioned above. I am no designer, I will leave colour and shape suggestions to the designers (@grarighe), for the rest I will make PRS :-)

**.focus style for buttons**
At the moment there is not so much different between a button with and without focus. But people using keyboard only it maybe hard to see of the button has focus or not.

**Links**
Links in the content should be indicated not by colour alone. Now the only difference is the colour.
This is important for colour blind people (8% of all men).
So make them stand out with an extra visual aid like underlining or bold or some neat design solution

Image: window in image shows the link for someone who can't distinguish colour.
![link-colour](https://cloud.githubusercontent.com/assets/2351479/20099542/53646ad2-a5b9-11e6-92ce-0098bbfe6fb2.png)

**Blockquote**
Use a <p> inside for the text.
```
<blockquote><p>Lorem ipsum dolor sit, amet elit. Aenean massa <strong>strong</strong>.  Nullam <a href="#">link</a> dictum felis eu pede pretium.</p>
</blockquote>
```
Specs: http://html5doctor.com/cite-and-blockquote-reloaded/

**The Accordion**
The accordion can be opened with keyboard only, but it's hard to see if the +/- gets focus or not.
Took me a white before I saw that. Focus doen't always go well when tabbing back and forth.

**Search bar**
Colour contrast placeholder needs improvement, especially as it's the only indicator about the action here.

**Tables**
Always give a table a caption and columns or rows a <th> depending on the structure.
If you don not want to show the captions or the <th>, use screen-reader-text or the equivalent.

Good read on screen-reader-text:
https://medium.com/@jessebeach/beware-smushed-off-screen-accessible-text-5952a4c2cbfe#.6f7yzeu6u

**Comment by mattheu** [comment]
Going to close this one. These issues refer to the old version.
